rfc-parser/docs/ModelDeviations.md

```diff
@@ -562,6 +600,44 @@ After:
 ![statements_to_indent_after.png](doc-images/statements_to_indent_after.png)
 ![indentable_string.png](doc-images/indentable_string.png)
 
+
+### String separation possibility for paths
```

Comment:
this looks like duplicate of the above. one should be cleared

Reply:
done